### PR TITLE
Add more ergonomics to tree

### PR DESCRIPTION
**Pull request by mgeplf**
* j/k for going up down, h/l for collapsing/opening node
* force certain tree order for special resource properties

### CODE REVIEW

**Comment by jdcourcol** [comment]
nice !

File: bluepyentity/app/explorer.py [hunk 1]
```diff
@@ -22,6 +23,21 @@ def _tree():
...
+
+    BINDINGS = [
+        Binding("enter", "select_cursor", "Select", show=False),
+        Binding("up", "cursor_up", "Cursor Up", show=False),
+        Binding("down", "cursor_down", "Cursor Down", show=False),
+        Binding("k", "cursor_up", "Cursor Up", show=False),
+        Binding("j", "cursor_down", "Cursor Down", show=False),
+        Binding("h", "select_cursor", "Cursor Up", show=False),
+        Binding("l", "select_cursor", "Cursor Down", show=False),
+        Binding("G", "action_scroll_end", "Goto end", show=False),
```
Comment (jdcourcol) [inline bluepyentity/app/explorer.py:37]: I did not get the "G" to work for me. I guess it is a textual issue or something with my terminal

**Review by jdcourcol — APPROVED** [review]
very nice

File: bluepyentity/app/explorer.py (hunk 1 above)
Comment (mgeplf) [reply bluepyentity/app/explorer.py]: yup, don't need the `action_` prefix in bindings, oops.

**Pull request merged**